docs/support.md

```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
 * In our [forum](https://xcp-ng.org/forum/category/12/xen-orchestra)
 * In our IRC - `#xen-orchestra` on `Freenode`
 
-However, there's no guarantee you will receive an answer and no guaranteed response time. If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.   
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free to use and change, supporting and developing it takes a lot of effort from a group of talented people, if you consider support business critical please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing). 
```

Comment:
We won't call ourselves like this :wink: Also we encourage people using in production to take support, not just only "business critical" missions.

Reply:
Won't call ourselves _like what_?

The deciding factor for any business to implement a solution should include "do I require support for this" if so, support is business critical and if support doesn't exist then the product is a Hobbyist solution at best. Since XOA is the solution that with a paid plan includes support this verbiage makes more sense.

Reply:
1. Like "talented people".
2. "Critical" means everything outside "critical" is OK from the sources. So it's like "critical" is even more a niche than "any" production usage. That's why I think this wording is not fit: every production load (ie in a company) makes sense. Hobbyist is clearly not related to "production", that's why "critical" word isn't needed here

Reply:
So you'd rather the PR say that the dev team of VatesFR isn't talented?

As for #2, using the word critical is anything that is a production work load, and specifically highlights in the English language that if:

**You don't have a support contract, then clearly you are OKAY with possibly severe outages and issues and that you accept these risks, and that you are willing to either fix the issue yourself or depend on the kindness of others.**

Reply:
1. I would like to avoid telling anything regarding our talent or not in the official documentation, because it's a purely subjective view
2. I don't know why we should insist about telling that, the initial wording seems pretty clear: in production (aka "real" env), please take support. Short, clear and concise. I don't understand why we should insist otherwise, as a hobbyist you know you don't run into production. As a company, it won't be serious to take a backup product without taking support or being able to debug the software.

Reply:
FYI, the rest of the PR is OK for me.

Reply:
IDK what you want for point 1, seems a bit insane to not speak positively about the people that contribute to this project. 

For number 2 that's just it though, you as the owner and supplier of support need to classify these installations "source installations" as hobbyist if they aren't paying for support so you have a way to separate the issues. 

The end user/admin may not do so, but you have to so you have a means of qualifying an installation as "supportable".

Reply:
1. Thing is: this is XO official documentation, we'd like to keep it neutral.
2. I don't think this require more clarification: you can clone something from GitHub, and there is a bug tracker at the very same place you clone it (if you want to do so), as any other OSS project on GH. Keep in mind this documentation is also used by XOA users, it's not targeting specifically "source users" (aka not XOA users). We don't have any issue "classifying" is GH tickets were made by hobbyist: all pro support comes into our internal support system. Our main issue was to debug issues coming from various platforms and installers, but this will be addressed in the future :)

Anyway, the goal of the original sentence was to just remind people using XO in production to use a validated solution (XOA) with support. Nothing more is needed IMHO.